tests/rptest/tests/rpk_cluster_test.py

```diff
@@ -86,6 +86,8 @@ def test_debug_bundle(self):
                 # dmidecode doesn't work in ducktape containers, ignore
                 # errors about it.
                 continue
+            if re.match('.* error querying .*\.ntp\..*', l):
```

Comment:
Sorry to be picky, but I really think we should be permissive only for the i/o timeout case.  Other NTP errors are still interesting.

Reply:
Not picky at all! I pushed the change.